net-print/cnijfilter-drivers/cnijfilter-drivers-3.80.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+MY_PN="${PN/-drivers/}"
+
+inherit eutils autotools flag-o-matic multilib
+
+DESCRIPTION="Canon InkJet Printer Driver for Linux (Pixus/Pixma-Series)."
+HOMEPAGE="http://support-sg.canon-asia.com/contents/SG/EN/0100469302.html"
+SRC_URI="http://gdlp01.c-wss.com/gds/3/0100004693/01/${MY_PN}-source-${PV}-1.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="${PV}"
```

Comment:
This package doesn't look like you can install multiple versions simultaneously, so slotting is wrong.

Reply:
Actually, yes, we should be able to install multiple versions of `cnijfilter-drivers`, as each printer model is supported only by a specific version. Only `cnijfilter` is common to all models and evolves through versions. See the discussion in the referenced bug. Ignored & for a next PR.

Reply:
If that is indeed the case, you should really add an appropriate comment. However, I should point out that such a layout will be terribly hard to use for the users.

Reply:
You are 100% right : this policy of Canon is really a nightmare for the users, and generate tons of discussions to find the right version for a given printer model, and to manage the obsolescence of that version. This is probably the reason why many guys try to merge all the versions in the latest (just look for projects with `cnijfilter` keyword on Github).

Reply:
Closed